lib/pure/htmlgen.nim

```diff
@@ -27,6 +27,21 @@
 ## MathML allows you to `draw advanced math on the web <https://developer.mozilla.org/en-US/docs/Web/MathML/Element/math#Examples>`_,
 ## `visually similar to Latex math. <https://developer.mozilla.org/en-US/docs/Web/MathML/Element/semantics#Example>`_
 ##
+## Embedding PDF
+## =============
```

Comment:
Offtopic. It's not our job to explain how `<embed>` works -- it's our job to explain how to produce the `<embed>` tag.